ecosystem/sep-0024.md

```diff
@@ -0,0 +1,688 @@
+## Preamble
+
+```
+SEP: 0024
+Title: Anchor/Client interoperability
```

Comment:
Should we adapt the title here to avoid confusion? Something like `Simplified Anchor/Client interoperability?`